bootstrap/config/kfctl_k8s_istio.yaml

```diff
@@ -1,150 +1,247 @@
 # This is the config to install Kubeflow on an existing k8s cluster.
-# If the cluster already has istio, comment out the istio install part (2 places) below.
+# If the cluster already has istio, comment out the istio install part below.
+
 apiVersion: kfdef.apps.kubeflow.org/v1alpha1
 kind: KfDef
 metadata:
-  name: demo
+  name: SET_APP_NAME
```

Comment:
Why not try to pick a more sensible name so if a user doesn't override its  still a decent name?